pywbem/_listener.py

```diff
@@ -798,6 +890,20 @@ def __init__(self, host, http_port=None, https_port=None,
 
             `None` means not to use a private key file. Setting up a port
             for HTTPS requires specifying a private key file.
+
+          max_ind_queue_size (:term:`integer`):
+            A positive integer which defines the maximum number of the received
+            indications that can be in the received indication queue.  If the
+            queue of received indications reaches this size, the queue is
+            blocked and the QueueFull exception is executed. No more
```

Comment:
Use the actual exception name with a link, instead of "QueueFull".

Reply:
Done